apps/frontend/src/app/3_organisms/BorrowLoan/components/AdjustLoanForm/hooks/useGetOriginationFee.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react';
+
+import { Decimal } from '@sovryn/utils';
+
+import { defaultChainId } from '../../../../../../config/chains';
+
+import { useLoadContract } from '../../../../../../hooks/useLoadContract';
+
+export const useGetOriginationFee = () => {
+  const [originationFee, setOriginationFee] = useState(Decimal.ZERO);
+  const contract = useLoadContract('protocol', 'protocol', defaultChainId);
+
+  useEffect(() => {
+    const fetchOriginationFee = async () => {
+      if (!contract) {
+        return;
+      }
+
+      const result = await contract.borrowingFeePercent();
```

Comment:
Can use caching so it would be called only once per block:
```suggestion
      const result = await asyncCall('protocol/borrowingFeePercent', () => contract.borrowingFeePercent());
```